src/versionedRouter.js

```diff
@@ -28,8 +28,6 @@ const { TRANSFORMER_METRIC } = require("./v0/util/constant");
 const networkHandlerFactory = require("./adapters/networkHandlerFactory");
 const profilingRouter = require("./routes/profiling");
 const destProxyRoutes = require("./routes/destinationProxy");
-
-require("dotenv").config();
```

Comment:
not required now?

Reply:
It's already imported [here](https://github.com/rudderlabs/rudder-transformer/blob/dff00b206615bb174863c9ad3eb3e7a879b34b2e/src/index.js#L4) in the entry point script.